src/Uno.UI/UI/Xaml/Controls/Primitives/PickerFlyoutBase.cs

```diff
@@ -4,12 +4,21 @@
 
 namespace Windows.UI.Xaml.Controls.Primitives
 {
-	public partial class PickerFlyoutBase : FlyoutBase
+	public abstract partial class PickerFlyoutBase : FlyoutBase
```

Comment:
Same here, this must not be abstract

Reply:
The constructor is protected, so it's the same...

Reply:
It's generally not a breaking change if no public methods can be called.